pkg/aws/helpers.go

```diff
@@ -402,9 +402,9 @@ func GetRolePath(roleARN string) (string, error) {
 	return path, nil
 }
 
-func GetRoleARN(accountID string, name string) string {
+func GetRoleARN(accountID string, name string, path string) string {
```

Comment:
I guess that if the user didn't provide a path, `path` equals an empty string.
If we decide to change the default value for the path to `/` it's going to affect this function as well.